astrobee/scripts/print_version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Print the software versions on the MLP, LLP, and HLP.
+
+if [ `hostname` != mlp ] ; then
+    echo "Must be run on mlp."
+    exit 1
+fi
+
+echo "=================== MLP ==================="
+./cpu_print_version.sh | sed 's/^/  /'
```

Comment:
Does this assume that we also need to be on the folder where the script is?

Reply:
Good point!